3rdparty/snmalloc/allocator.cpp

```diff
@@ -20,13 +20,15 @@ void* ThreadAllocUntyped::get()
 } // namespace snmalloc
 
 #define OPEN_ENCLAVE
+#define SNMALLOC_SGX
 #define USE_RESERVE_MULTIPLE 1
 #define IS_ADDRESS_SPACE_CONSTRAINED
 #define SNMALLOC_EXTERNAL_THREAD_ALLOC
 #define SNMALLOC_NAME_MANGLE(a) oe_allocator_##a
 
 // Enable the Open Enclave PAL(Platform Abstraction Layer) for Open Enclave,
 // see pal_open_enclave.h for details
+#include "./snmalloc/src/ds/address.h"
```

Comment:
This should be included by malloc.cc, will fix upstream.

Reply:
The PalOpenEnclave file should include address.h

Reply:
https://github.com/microsoft/snmalloc/pull/205

Reply:
@mjp41 It seems address.h ought to be included by specifying the relative path from pal_openenclave.h. The compiler could not find "ds/address.h". I worked around this by adding snmalloc/src as an include directory when building snmalloc for OE.